src/types.ts

```diff
@@ -137,6 +137,13 @@ export enum IconColor {
   WHITE = 'text-white',
 }
 
+export enum Opacity {
+  READ = 'opacity-50',
```

Comment:
What's the reasoning behind the name?

Reply:
That's what we currently set a Notification Row as when it is `READ` during the delay notification state change.

It was what I had used, too, in the draft PR for seeing read notifications.

Do you have a better name suggestion?